docker/openemr/7.0.1/openemr.sh

```diff
@@ -391,6 +391,13 @@ fi
 if [ "$XDEBUG_IDE_KEY" != "" ] ||
    [ "$XDEBUG_ON" == 1 ]; then
    sh xdebug.sh
+else
+   # Configure opcache jit if Xdebug is not being used
+   if [ ! -f /etc/php-opcache-jit-configured ]; then
+      echo "opcache.jit_buffer_size=100M" >> /etc/php81/php.ini
+      echo "opcache.jit=tracing" >> /etc/php81/php.ini
```

Comment:
the line above not needed since this is the default setting: https://www.php.net/manual/en/opcache.configuration.php
(note that the opcache.jit_buffer_size set to 100M is needed though since is not default)

Reply:
Hang on, if it's not needed because it's default then it's *default on* which means that the real problem is that's *not off* on the other branch and we need to fix that instead.

Reply:
yep, ended up doing that in the if/else in comment below (turns it off when xdebug is on :) )